lib/ui/compositing/scene_builder.cc

```diff
@@ -57,56 +57,78 @@ void SceneBuilder::RegisterNatives(tonic::DartLibraryNatives* natives) {
 }
 
 SceneBuilder::SceneBuilder()
-    : m_currentLayer(nullptr), m_currentRasterizerTracingThreshold(0) {}
+    : m_currentLayer(nullptr), m_currentRasterizerTracingThreshold(0) {
+  m_cullRects.push(SkRect::MakeLargest());
```

Comment:
I think I will do that in a separate PR. It looks like the RenderView SceneBuilder's clip has to be kept in sync with its view configuration ala ReplaceRootLayer. Maybe we need ReplaceRootLayers?